client/app/assets/styles/variables.js

```diff
@@ -139,8 +139,10 @@ module.exports = {
 
   '--MenuItem_borderColor': topbarBorderColor,
   '--MenuItem_backgroundColorHover': backgroundLightColorHover,
-  '--MenuItem_paddingTopbarVertical': paddingTopbarVertical,
-  '--MenuItem_paddingTopbarHorizontal': paddingTopbarHorizontal,
+  '--MenuItem_paddingTopbarVertical': '0.92857em',
+  '--MenuItem_paddingTopbarHorizontal': '1.7142857em',
+  '--MenuItem_paddingOffScreenVertical': '0.58823529411em',
+  '--MenuItem_paddingOffScreenHorizontal': '1.41176470588em',
```

Comment:
Yeah, values come from conversions (some padding / font-size). The problem is that since we can't use rems here, I need to use ems, and font-sizes varies a lot on Janne's design.  I'll change this to pixels.